services/QuillLMS/spec/models/clever_classroom_user_spec.rb

```diff
@@ -25,14 +25,15 @@
 require 'rails_helper'
 
 RSpec.describe CleverClassroomUser, type: :model do
-  let(:clever_classroom_id) { '67890' }
-  let(:clever_user_id) { '12345' }
+  subject { create(:clever_classroom_user)}
 
-  subject do
-    create(:clever_classroom_user, classroom_external_id: clever_classroom_id, user_external_id: clever_user_id)
-  end
+  it_behaves_like 'a provider classroom user'
```

Comment:
On the one hand, I love this short-cut.  On the other hand, I hate the magic or a named string like this since it can be hard to infer the file location for an example like this from the string.  Still, better than repeating a bunch of stuff.

Reply:
I totally agree.  I've always found these 'string phrases' introduce a bizarre coupling to the tests.